### ISSUE TITLE
User mailboxes

### ISSUE THREAD

**Issue by strugee** [osuosl-cookbooks/proj-seagl#8]
Forking discussion from #4. From https://github.com/osuosl-cookbooks/proj-seagl/issues/4#issuecomment-1413089833:

> For context, here are our [email needs], as far as I understand them:
> 
> * We ideally would have individual @seagl.org mailboxes for folks, though we could possibly work around the lack of this with some alias forwarding + rules in folks' personal inboxes
> * We _need_ some kind of sharing mechanism for certain email addresses (think coc@seagl.org, program@seagl.org etc.) where multiple people will want to respond to emails as they come in
> * We _need_ people to be able to send from @seagl.org mailing addresses, including the above shared addresses
> * We have pretty standard mailing list needs, though we'd _ideally_ like to be able to administer lists ourselves

What support does OSUOSL offer here? Looking at https://osuosl.org/services/hosting/details/#e-mail it seems like it may make the most sense to run our own mail systems and use OSUOSL's relays for virus and spam protection? Though ideally we would like to avoid running our own stuff as much as possible, if there's something else not mentioned on that page.

**Comment by ramereth** [comment]
We have been working on getting a more centralized mailbox hosted solution going, but it's not quite ready yet for projects. Do you have an idea of when you would like this ready?

**Comment by strugee** [comment]
We probably want it ASAP since we're currently getting billed by Google for Work and this is a big blocker for moving off it. That being said maybe we could spin up our own mail servers on OpenStack (@altsalt I know had a solution in mind) and we can coordinate moving data into OSUOSL's solution whenever that's ready?

**Comment by altsalt** [comment]
Yes, I've had my eyes on [Mailu](https://mailu.io/) for a while now, but less stuff that we are running that isn't conference-event specific, the better.

**Comment by strugee** [comment]
@ramereth any objections to us just spinning up Mailu on an OpenStack VM?

**Comment by ramereth** [comment]
@strugee nope not at all, Given our current queue, I'm not sure when we'll get to this going on our end. Do you need anything from us to do that?

**Comment by strugee** [comment]
No problem, we'll work on setting this up. I don't think we need anything special though I'll let you know if we do.
